src/rocksdb_options.rs

```diff
@@ -153,6 +154,8 @@ pub struct RateLimiter {
     inner: *mut DBRateLimiter,
 }
 
+unsafe impl Send for RateLimiter { }
```

Comment:
maybe we can also implement Sync here if RocksDB RaftLimiter is thread-safe.

Reply:
sure